### PR TITLE
ru_reviews dataset adding

### PR DESCRIPTION
**Pull request by darshan-gandhi**
RuReviews: An Automatically Annotated Sentiment Analysis Dataset for Product Reviews in Russian

### CODE REVIEW

**Comment by ijaz314** [comment]
Hii please follow me

**Comment by darshan-gandhi** [comment]
Hi @lhoestq  

I have added the readme as well 

Please do have a look at it when suitable

File: datasets/rureviews/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+YAML tags:
```
Comment (yjernite) [inline datasets/rureviews/README.md:2]:
```suggestion
```

**Comment by yjernite** [comment]
Chatted with @darshan-gandhi on Slack about parsing examples into a separate text and sentiment field

File: datasets/rureviews/rureviews.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+"""
+
+
+_TRAIN_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/sismetanin/rureviews/master/women-clothing-accessories.3-class.balanced.csv"
+)
+
+
+class AppReviews(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/rureviews/rureviews.py:55]:
```suggestion
class Rureviews(datasets.GeneratorBasedBuilder):
```

File: datasets/rureviews/rureviews.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        train_path = dl_manager.download_and_extract(_TRAIN_DOWNLOAD_URL)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"filepath": train_path}),
+        ]
+
+    def _generate_examples(self, filepath):
+        """Generate Distaster Response Messages examples."""
+        with open(filepath, encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(
+                csv_file, quotechar='"', delimiter=",", quoting=csv.QUOTE_ALL, skipinitialspace=True
```
Comment (lhoestq) [inline datasets/rureviews/rureviews.py:81]: The delimiter is `\t` here if I understand correctly.
Changing the delimiter when creating the csv reader would avoid having to use these lines later 
```
                row = [s for s in row if "\t" in s]
                row = row[0].split("\t")
```

File: datasets/rureviews/rureviews.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+class AppReviews(datasets.GeneratorBasedBuilder):
+    """RuReviews: An Automatically Annotated Sentiment Analysis Dataset for Product Reviews in Russian"""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "review": datasets.Value("string"),
+                    "sentiment": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/rureviews/rureviews.py:64]: Can you use the ClassLabel feature type here ?

for example
```
"sentiment": datasets.ClassLabel(names=["negative", "neutral", "positive"])
```

File: datasets/rureviews/rureviews.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            next(csv_reader, None)
+
+            for id_, row in enumerate(csv_reader):
+                row = [s for s in row if "\t" in s]
+                row = row[0].split("\t")
+                if len(row) == 2:
+                    review, sentiment = row
+                elif len(row) == 3 or len(row) == 4:
+                    review = " ".join(row[:-1])
+                    sentiment = row[-1]
```
Comment (lhoestq) [inline datasets/rureviews/rureviews.py:92]: In the dataset csv there's a typo for the `neutral` sentiment. Can you fix it please ?
```suggestion
                    sentiment = row[-1]
                    if sentiment == "neautral":
                        sentiment = "neutral"
```

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one !

I left a few comments.
After the changes you'll need to regenerate the dataset_infos.json file

Also you dummy data test is failing because in your dummy csv file you're separating the columns with `,` instead of `\t`. Can you fix the dummy data csv file please ?

**Comment by albertvillanova** [comment]
Thanks for your contribution, @darshan-gandhi. Are you still interested in adding this dataset?

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

We would suggest you create this dataset there. Please, feel free to tell us if you need some help.

**Pull request closed without merging**